Sources/BraintreeVenmo/BTVenmoAppSwitchRedirectURL.swift

```diff
@@ -57,6 +55,16 @@ struct BTVenmoAppSwitchRedirectURL {
             "braintree_sdk_data": base64EncodedBraintreeData ?? "",
             "customerClient": "MOBILE_APP"
         ]
+
+        if let universalLink {
+            queryParameters["x-success"] = universalLink.absoluteString + "/success"
+            queryParameters["x-error"] = universalLink.absoluteString + "/error"
+            queryParameters["x-cancel"] = universalLink.absoluteString + "/cancel"
```

Comment:
Can we use URLComponents, or some other Apple built-in function, to do a path append? This way, we don't care if a merchant passes in a URL with an end ing slash `/` or not?